mapeditor/inspector/inspector.cpp

```diff
@@ -612,6 +614,9 @@ void Inspector::setProperty(CGTownInstance * o, const QString & key, const QVari
 	if(key == QObject::tr("Town name"))
 		o->setNameTextId(mapRegisterLocalizedString("map", *controller.map(),
 			TextIdentifier("town", o->instanceName, "name"), value.toString().toStdString()));
+
+	if(key == QObject::tr("Same as player"))
```

Comment:
- comparing with a user-visible string is not a great idea, it should rather be some enum or internal string constant
- `else if`

Reply:
Would require fixing whole file, possibly in separate PR

Reply:
Ok, that's fine